### ISSUE TITLE
Outdated documentation on QuickStart page?

### ISSUE THREAD

**Issue by astrochun** [globus/globus-cli#480]
I recently installed globus-cli and went through the instructions here:
https://docs.globus.org/cli/quickstart/

I've successfully logged in.  The QuickStart instructions say:
```
You can check that you can correctly authenticate to the Globus APIs with two quick commands.
First, check that you can access Globus Auth:

$ globus get-identities 'go@globusid.org'
ID                                   | Full Name      | Username        | Organization | Email Address
------------------------------------ | -------------- | --------------- | ------------ | ------------------
c699d42e-d274-11e5-bf75-1fc5bf53bb24 | www.globus.org | go@globusid.org | Globus       | noreply@globus.org
```

However, that is not what I see.  I only see the ID, not the remaining columns:

```
$ globus get-identities 'go@globusid.org'
c699d42e-d274-11e5-bf75-1fc5bf53bb24
```

I'm running:
```
$ globus version
Installed Version: 1.11.0
Latest Version:    1.11.0
```

The information is also outdated (columns were swapped) in this example:
```
$ globus endpoint search 'Globus Tutorial Endpoint' \
     --filter-owner-id 'c699d42e-d274-11e5-bf75-1fc5bf53bb24'
ID                                   | Owner           | Display Name               
------------------------------------ | --------------- | ---------------------------
ddb59aef-6d04-11e5-ba46-22000b92c6ec | go@globusid.org | Globus Tutorial Endpoint 1 
ddb59af0-6d04-11e5-ba46-22000b92c6ec | go@globusid.org | Globus Tutorial Endpoint 2 
cf9bcaa5-6d04-11e5-ba46-22000b92c6ec | go@globusid.org | Globus S3 Tutorial Endpoint
```

**Comment by sirosen** [comment]
Thanks for reporting these items!
I'll do a review of the documentation site content and get it updated to match reality.

Since we don't guarantee that the text format stays the same version-to-version, I think this is just a bit of drift. We added `--verbose` to `globus get-identities` to get the full output, at which point we also changed the non-verbose output.
If I recall correctly, the `globus get-identities` change was made to better support usage like `my_id="$(globus get-identities sirosen@uchicago.edu)"`. I can't recall why we would have reordered the columns of endpoint search output, but, regardless, the right thing to do is still just to update our docs to be accurate.
